content/v2/tlds.markdown

```diff
@@ -47,6 +47,16 @@ List all TLDs.
 <%= pretty_print_fixture("/listTlds/success.http") %>
 ~~~
 
+### Sorting
+
+For general information about sorting, please refer to the [main guide](/v2/#sorting).
+
+Name | Description
+-----|------------
+`tld` | Sort TLDs by tld
```

Comment:
I know this is how we named the field but it sounds confusing... _Short the Foos by foo_.  Maybe _suffix_ onr _name_ could have been a better name.